ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -1931,19 +2186,65 @@ const triggerSet: TriggerSet<Data> = {
         // in outputStrings; see #266 for more info
         let towerSoakStr = output['unknown']!();
         let cannonBaitStr = output['unknown']!();
+        let cannonBaitSpots = undefined;
 
         if (data.sunriseTowerSpots !== undefined) {
-          towerSoakStr = output[data.sunriseTowerSpots]!();
-          cannonBaitStr = data.sunriseTowerSpots === 'northSouth'
-            ? output.eastWest!()
-            : output.northSouth!();
+          if (data.triggerSetConfig.sunrise === 'snakePrio') {
+            if (data.sunriseTowerSpots === 'northSouth') {
+              towerSoakStr = data.role === 'dps'
+                ? output.northTower!()
+                : output.southTower!();
+            } else {
+              towerSoakStr = data.role === 'dps'
+                ? output.eastTower!()
+                : output.westTower!();
+            }
+          } else {
+            towerSoakStr = output[data.sunriseTowerSpots]!();
+          }
+          if (data.triggerSetConfig.sunriseUptime) {
+            cannonBaitSpots = data.sunriseTowerSpots;
+            cannonBaitStr = data.sunriseTowerSpots === 'northSouth'
+              ? output.northSouth!()
+              : output.eastWest!();
+          } else {
+            cannonBaitSpots = data.sunriseTowerSpots === 'northSouth'
+              ? 'eastWest'
+              : 'northSouth';
+            cannonBaitStr = data.sunriseTowerSpots === 'northSouth'
+              ? output.eastWest!()
+              : output.northSouth!();
+          }
         }
 
         if (task === 'yellowShort' || task === 'blueShort') {
           const cannonLocs = task === 'yellowShort' ? blueCannons : yellowCannons;
+
+          if (data.triggerSetConfig.sunrise === 'snakePrio') {
+            const dpsPrio: DirectionOutputIntercard[] = ['dirNE', 'dirSE', 'dirSW', 'dirNW'];
+            const supPrio: DirectionOutputIntercard[] = ['dirNW', 'dirSW', 'dirSE', 'dirNE'];
```

Comment:
```suggestion
            const dpsPrio: DirectionOutputIntercard[] = ['dirNE', 'dirSE', 'dirSW'];
            const supPrio: DirectionOutputIntercard[] = ['dirNW', 'dirSW', 'dirSE'];
```

With snake prio, I don't think the fourth spot will ever be used, right?

Reply:
Nope. I can remove it.